pom.xml

```diff
@@ -286,7 +287,7 @@
 								<argument>{project.build.directory}/${project.artifactId}-dev-${project.version}-fat.jar
 								</argument>
 								<argument>-c</argument>
-								<argument>./configs/config-dev.json</argument>
+								<argument>./secrets/credentials/all-verticles-configs/${config-dev.file}</argument>
```

Comment:
It's kind of odd to have config files inside secrets/credentials folder
Maybe we can think of better names.

Reply:
yes, we can think of better name, but basically the configs contain the credentials. if we can separate configuration and credentials, but that is tedious task. 

Have named the the prefix directories as "secrets/credentials" to really point out its sensitive and shouldn't be committed.

Reply:
You can just keep it as config and credentials are in it. 
Basically configs will be obtained through docker secrets right..?

Reply:
Docker secrets is only possible with docker stack and stack you->need a docker swarm (so a dockwer swarm in dev env also). But docker creates the secrets from files, so I am positioning this files in the secrets directory. See [here](https://github.com/datakaveri/iudx-deployment/tree/rs-patches/single-node/resource-server/apiserver). 
![Screenshot from 2021-03-03 13-29-44](https://user-images.githubusercontent.com/29863835/109773089-cc1c3680-7c24-11eb-9a58-197c4896d26c.png)